internal/api/scim/integration_test/users_replace_test.go

```diff
@@ -286,7 +286,6 @@ func TestReplaceUser(t *testing.T) {
 			}, retryDuration, tick)
 		})
 	}
-
```

Comment:
I would personally remove format changes in untouched files, could lead to linting problems.